### PR TITLE
Frontend Rough Draft

### PR DESCRIPTION
**Pull request by lmtroper**
# Description

I built out the main pages discussed for the frontend which include a login/home page. Once logged in, there's a page for a user's subreddits that they're subscribed to, a page to explore (i.e., search) other subreddits, and one page for their profile.

To start the app, `cd` into folder and run: `npm run dev`

There is no functionality atm so you can just click on Login to see the other pages. 

This is also just a drafted version, could benefit by a bit more design work (lol) but just requesting to merge this now so we aren't waiting for it in order to start connecting it to backend.

### CODE REVIEW

**Comment by Adelpzk** [comment]
image
Can we make these the same colour as the log in button?

**Comment by Adelpzk** [comment]
image
Any way to make these orange with black check marks? :D

**Comment by Adelpzk** [comment]
image
Can we find a better shade of orange for this? 
and can we add some sort of hover effect to the side bar buttons

Beside the few comments everything looks really beautiful. Love the UI!

File: next-frontend/src/components/LoginCard.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            <div className="mb-1">
+                <input className="login" type="text" placeholder={"Password"} />
+            </div>
+            <a className="flex justify-end text-sm b-50">
+                <span>Forgot password?</span>
+            </a>
+            <Link href="/subreddits">
+                <button className="btn email-login-btn">Log in</button>
+            </Link>
+            <div class="line-or-line">
```
Comment (aniqp) [inline next-frontend/src/components/LoginCard.js]: Had to change `class` to `className` for it to compile.

**Pull request merged**